src/output-json-dns.c

```diff
@@ -187,66 +189,21 @@ typedef enum {
 static struct {
     const char *config_rrtype;
     uint64_t flags;
-} dns_rrtype_fields[] = {
-   { "a", LOG_A },
-   { "ns", LOG_NS },
-   { "md", LOG_MD },
-   { "mf", LOG_MF },
-   { "cname", LOG_CNAME },
-   { "soa", LOG_SOA },
-   { "mb", LOG_MB },
-   { "mg", LOG_MG },
-   { "mr", LOG_MR },
-   { "null", LOG_NULL },
-   { "wks", LOG_WKS },
-   { "ptr", LOG_PTR },
-   { "hinfo", LOG_HINFO },
-   { "minfo", LOG_MINFO },
-   { "mx", LOG_MX },
-   { "txt", LOG_TXT },
-   { "rp", LOG_RP },
-   { "afsdb", LOG_AFSDB },
-   { "x25", LOG_X25 },
-   { "isdn", LOG_ISDN },
-   { "rt", LOG_RT },
-   { "nsap", LOG_NSAP },
-   { "nsapptr", LOG_NSAPPTR },
-   { "sig", LOG_SIG },
-   { "key", LOG_KEY },
-   { "px", LOG_PX },
-   { "gpos", LOG_GPOS },
-   { "aaaa", LOG_AAAA },
-   { "loc", LOG_LOC },
-   { "nxt", LOG_NXT },
-   { "srv", LOG_SRV },
-   { "atma", LOG_ATMA },
-   { "naptr", LOG_NAPTR },
-   { "kx", LOG_KX },
-   { "cert", LOG_CERT },
-   { "a6", LOG_A6 },
-   { "dname", LOG_DNAME },
-   { "opt", LOG_OPT },
-   { "apl", LOG_APL },
-   { "ds", LOG_DS },
-   { "sshfp", LOG_SSHFP },
-   { "ipseckey", LOG_IPSECKEY },
-   { "rrsig", LOG_RRSIG },
-   { "nsec", LOG_NSEC },
-   { "dnskey", LOG_DNSKEY },
-   { "dhcid", LOG_DHCID },
-   { "nsec3", LOG_NSEC3 },
-   { "nsec3param", LOG_NSEC3PARAM },
-   { "tlsa", LOG_TLSA },
-   { "hip", LOG_HIP },
-   { "cds", LOG_CDS },
-   { "cdnskey", LOG_CDNSKEY },
-   { "spf", LOG_SPF },
-   { "tkey", LOG_TKEY },
-   { "tsig", LOG_TSIG },
-   { "maila", LOG_MAILA },
-   { "any", LOG_ANY },
-   { "uri", LOG_URI }
-};
+} dns_rrtype_fields[] = { { "a", LOG_A }, { "ns", LOG_NS }, { "md", LOG_MD }, { "mf", LOG_MF },
+    { "cname", LOG_CNAME }, { "soa", LOG_SOA }, { "mb", LOG_MB }, { "mg", LOG_MG },
+    { "mr", LOG_MR }, { "null", LOG_NULL }, { "wks", LOG_WKS }, { "ptr", LOG_PTR },
+    { "hinfo", LOG_HINFO }, { "minfo", LOG_MINFO }, { "mx", LOG_MX }, { "txt", LOG_TXT },
+    { "rp", LOG_RP }, { "afsdb", LOG_AFSDB }, { "x25", LOG_X25 }, { "isdn", LOG_ISDN },
+    { "rt", LOG_RT }, { "nsap", LOG_NSAP }, { "nsapptr", LOG_NSAPPTR }, { "sig", LOG_SIG },
+    { "key", LOG_KEY }, { "px", LOG_PX }, { "gpos", LOG_GPOS }, { "aaaa", LOG_AAAA },
+    { "loc", LOG_LOC }, { "nxt", LOG_NXT }, { "srv", LOG_SRV }, { "atma", LOG_ATMA },
+    { "naptr", LOG_NAPTR }, { "kx", LOG_KX }, { "cert", LOG_CERT }, { "a6", LOG_A6 },
+    { "dname", LOG_DNAME }, { "opt", LOG_OPT }, { "apl", LOG_APL }, { "ds", LOG_DS },
+    { "sshfp", LOG_SSHFP }, { "ipseckey", LOG_IPSECKEY }, { "rrsig", LOG_RRSIG },
+    { "nsec", LOG_NSEC }, { "dnskey", LOG_DNSKEY }, { "dhcid", LOG_DHCID }, { "nsec3", LOG_NSEC3 },
+    { "nsec3param", LOG_NSEC3PARAM }, { "tlsa", LOG_TLSA }, { "hip", LOG_HIP }, { "cds", LOG_CDS },
+    { "cdnskey", LOG_CDNSKEY }, { "https", LOG_HTTPS }, { "spf", LOG_SPF }, { "tkey", LOG_TKEY },
+    { "tsig", LOG_TSIG }, { "maila", LOG_MAILA }, { "any", LOG_ANY }, { "uri", LOG_URI } };
```

Comment:
Was this clang-format's doing? 😢

Reply:
Yes. I believe it looked nicer before. But wasn't sure if I should revert back or not.

Reply:
yes please revert this, then wrap it in:
```c
// clang-format off
... the table ...
// clang-format on
```

Reply:
done